shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -538,7 +570,7 @@ public void onStart() {
     new Handler().post(new Runnable() {
       @Override
       public void run() {
-        flutterView.attachToFlutterEngine(flutterEngine);
+        flutterView.attachToFlutterEngine(flutterEngine, platformViewsController);
```

Comment:
I'm surprised something needed to be done to the platform view controller multiple times

Reply:
I'm not sure what you're referring to?

Reply:
Talked offline, resolving old comments.